package.json

```diff
@@ -8,17 +8,17 @@
     "dist"
   ],
   "scripts": {
-    "build": "webpack --config webpack.config.js && npm run api-report",
+    "build": "webpack --config webpack.config.js && npm run api-report && npm run prune",
+    "api-report": "api-extractor run --local --verbose",
     "build:proto": "protoc --js_out=import_style=commonjs:. --grpc-web_out=import_style=commonjs+dts,mode=grpcwebtext:. ./src/api/yorkie.proto",
-    "build:docs": "api-documenter markdown --input temp --output docs",
+    "prune": "ts-node-script ./script/prune-dts.ts --input ./dist/yorkie-js-sdk.d.ts --output ./dist/yorkie-js-sdk.d.ts",
+    "predoc": "ts-node-script ./script/api-extractor.ts",
+    "build:docs": "npm run predoc && api-documenter markdown --input temp --output docs",
     "start": "webpack-dev-server --watch --config webpack.dev.config.js",
     "test": "karma start karma.conf.js --single-run",
     "test:yorkie.dev": "karma start karma.conf.js --single-run --testRPCAddr=https://yorkie.dev/api",
     "test:watch": "karma start karma.conf.js",
-    "lint": "eslint . --fix --max-warnings=0 --ext .ts",
-    "prepare": "npm run build",
```

Comment:
Is there a reason we removed `prepare`? Previously, if the user installs yorkie-js-sdk with npm install, `prepare` is preparing the bundle.

Reply:
I took a look this script https://github.com/yorkie-team/yorkie-js-sdk/blob/main/.github/workflows/npm-publish.yml
I think after we run `npm run build`, I think `yorkie-js-sdk.js`, `yorkie-js-sdk.d.ts` are generated. isn't it? 🤔

Reply:
This script was previously added when [removing the /dist folder from the source code](https://github.com/yorkie-team/yorkie-js-sdk/commit/098090f2319c0af4ea77226301f36d447bf2c996). I remember that without the script there was no dist file after npm install from the specific branch directly(e.g. `npm install https://github.com/yorkie-team/yorkie-js-sdk#main`). 🤔

https://stackoverflow.com/questions/17509669/how-to-install-an-npm-package-from-github-directly

Reply:
I see thank you for letting me know!